numpy/polynomial/__init__.py

```diff
@@ -31,10 +31,10 @@
 The convenience classes are the preferred interface for the `~numpy.polynomial`
 package, and are available from the `numpy.polynomial` namespace.
 This eliminates the need to
-navigate to the corresponding submodules, e.g. ``np.polynomial.Polynomial``
-or ``np.polynomial.Chebyshev`` instead of
-``np.polynomial.polynomial.Polynomial`` or 
-``np.polynomial.chebyshev.Chebyshev``, respectively.
+navigate to the corresponding submodules, e.g. `numpy.polynomial.Polynomial`
+or `numpy.polynomial.Chebyshev` instead of
+`numpy.polynomial.polynomial.Polynomial` or 
+`numpy.polynomial.chebyshev.Chebyshev`, respectively.
```

Comment:
Do all four of these links actually work in the docs? I'd expect only two of them to work.

Reply:
Do any of the CI tests build docs that can be viewed?

Reply:
Yes, here's the broken links as predicted: https://18124-908607-gh.circle-artifacts.com/0/doc/build/html/reference/routines.polynomials.package.html?highlight=polynomial#module-numpy.polynomial